lib/cli/file-set-pipeline/traverse.js

```diff
@@ -27,6 +27,10 @@ var Traverser = require('../traverser');
  * @param {CLI} cli
  */
 function traverse(cli) {
+    if (cli.changedFilePath) {
+        return;
```

Comment:
@wooorm So here's what I'm talking about. Try the following.

Test case 1:
- Run `mdast -w readme.md`, with no errors in `readme.md`.
- Make an error in `readme.md`. No warnings show up. This is a bug.

Test case 2:
- Before you run mdast, make a line in `readme.md` longer than 80 characters. Make it like 90 characters.
- Run `mdast -w readme.md`, with the error still in `readme.md`.
- You get a warning upon mdast's load. This is correct.
- Delete one character at the end of that 90-character line. You'll notice the warnings from prior runs pile up. This is a bug.